### DISCUSSION TITLE
How do I modify the system prompt while using plugins?

### DISCUSSION THREAD

**Discussion by grexzen** [danny-avila/LibreChat#1079] [Q&A] [answered]
The prompt is greyed out in the UI, and I need to change it because my plugin works best when GPT knows what to do in general. 
Also w/o it, we get the typical tons of tokens for silly stuff like "As a blah...".

If you point me to the right folder or file, I can do it.

Thanks!

**Comment by danny-avila** [comment, marked as answer]
if it's for plugins using functions, you could try in the following file: `api/app/clients/agents/Functions/initializeFunctionsAgent.js`
